test/unittests/commands/excel2xml/test_excel2xml_lib.py

```diff
@@ -124,7 +124,7 @@ def run_test(
             # a <text> has the additional attribute encoding="utf8" (the other encoding, xml, is tested in the caller)
             xml_expected = regex.sub(
                 r"<text (permissions=\".+?\")( comment=\".+?\")?",
-                '<text \\1\\2 encoding="utf8"',
+                '<text \\1 encoding="utf8"\\2',
```

Comment:
I excuse myself for the code & the tests that my past self wrote 🙈  They are so ugly that I had to adapt the order of the attributes here. If I have too much time some day, I will refactor all this :-)